diff -r -U 3 minecraft_server\net\minecraft\world\storage\ThreadedFileIOBase.java minecraft_server_patched\net\minecraft\world\storage\ThreadedFileIOBase.java
--- minecraft_server\net\minecraft\world\storage\ThreadedFileIOBase.java
+++ minecraft_server_patched\net\minecraft\world\storage\ThreadedFileIOBase.java
@@ -21,6 +21,7 @@
    }
 
    public void run() {
+     while(true)
       this.func_75736_b();
    }
 
